examples/child_window.rs

```diff
@@ -46,7 +46,7 @@ fn main() -> Result<(), impl std::error::Error> {
 
     println!("parent window: {parent_window:?})");
 
-    event_loop.run(move |event: Event<'_, ()>, event_loop, control_flow| {
+    event_loop.run(move |event: Event<()>, event_loop, control_flow| {
```

Comment:
Do we have no examples that use `ScaleFactorChanged`?

That seems like an oversight, and makes it hard for us to judge how the API will be used by users.

Reply:
I guess the main case is to write when changing to some size the user wants so you don't have a window jumping in sizes, but not sure that we really want to demonstrate it.